ui/components/app/simulation-details/simulation-details.tsx

```diff
@@ -47,13 +47,14 @@ const ErrorContent: React.FC<{ error: SimulationError }> = ({ error }) => {
   const t = useI18nContext();
 
   function getMessage() {
-    if (error.isReverted) {
+    if (
+      error.isReverted ||
+      error.message?.includes('insufficient funds for gas')
+    ) {
       return t('simulationDetailsTransactionReverted');
     }
-    if (error.message) {
-      if (error.message.includes('Chain is not supported')) {
-        return t('simulationDetailsChainNotSupported');
-      }
+    if (error.message?.includes('Chain is not supported')) {
```

Comment:
I think Saya has confirmed the current goal is to collapse the entire component if the chain is not supported so I assume we can just return `null` at the root instead.

Reply:
Done.